BGL/include/CGAL/boost/graph/iterator.h

```diff
@@ -161,7 +161,26 @@ struct Face {
     return face(h,*g);
   }
 };
+template <typename G>
+struct OppositeFace {
```

Comment:
The naming of that class template does not following the naming guidelines of CGAL. Is that on purpose? Do you follow other naming conventions, based on the BGL?